onnxruntime/core/optimizer/conv_add_fusion.cc

```diff
@@ -16,10 +16,16 @@ Status ConvAddFusion::Apply(Graph& graph, Node& node, RewriteRuleEffect& modifie
   const auto& add_inputs = add_node.InputDefs();
 
   const ONNX_NAMESPACE::TensorProto* conv_W_tensor_proto = nullptr;
-  graph.GetInitializedTensor(conv_inputs[1]->Name(), conv_W_tensor_proto);
+  bool rc = graph.GetInitializedTensor(conv_inputs[1]->Name(), conv_W_tensor_proto);
+  if (!rc) {
+    return Status::OK();
+  }
```

Comment:
* Instead of adding the bool, maybe we can simply check that the conv_W_tensor_proto is not a nullptr? Same below.
* There is a third place in this file that call the GetInitializedTensor.
* We should probably do a similar fix for the ConvMulFusion and the ConvBNFusion?
* In general, do we consider it a valid model if it uses an initializer that is not present?

Reply:
* GetInitializedTensor was implemented before this PR. Since GetInitializedTensor already returns an error code, it makes sense to check if the function succeeded or not and bail out if it failed. That's the contract of the function. Why would you proceed if the function you're calling failed in the first place!
* Sure, I can fix the third place as well.
* Sure, will fix in ConvMulFusion and ConvBNFusion.
* In this case, there was no initializer used. The W was an input.